client/jetpack-cloud/components/layout/style.scss

```diff
@@ -0,0 +1,154 @@
+.is-jetpack-new-navigation {
```

Comment:
The majority of the styling used in this section has been replicated from the original `partner-portal/layout` component.